packages/database/src/migrations/20210119053017-SomoaCovidCustomisedRawDataDownload-modifies-data.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+import { insertObject, addReportToGroups } from '../utilities/migration';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const dashboardReportCode = 'WS_Covid_Samoa_Country';
+const surveyCode = 'SC1QMIA';
+const newDashboardReport = {
+  id: 'Samoa_Covid_Customised_Raw_Data_Download',
+  dataBuilder: 'surveyDataExport',
+  dataBuilderConfig: {
+    surveys: [
+      {
+        code: surveyCode,
+        name: 'Initial Quarantine Monitoring',
+      },
+    ],
+    exportDataBuilder: {
+      dataBuilder: 'rawDataValues',
+      dataBuilderConfig: {
+        skipHeader: false,
+        surveysConfig: {
+          [surveyCode]: {
+            entityAggregation: {
+              dataSourceEntityType: 'case',
+            },
+            entityAnswerIdToName: { dataElementCodes: ['QMIA029'] },
```

Comment:
We can use a flatter option here:

```suggestion
            entityIdToNameElements: ['QMIA029'],
```

Reply:
Can I leave this config as an object? I was thinking in the future we might need more configs related to the entity name, like adding spaces at front and behind, color or font size. But I'm happy to make it simple and sharp.

Reply:
I had similar concerns when working with data builders... I found out that most of the time, configs were better being simple rather than allowing space for something that never came in the future.

This is a case by case decision though, happy to leave it to you here